test/src/unit-backwards_compat.cc

```diff
@@ -1247,17 +1247,6 @@ TEST_CASE(
     "Backwards compatibility: Upgrades an array of older version and "
     "write/read it",
     "[backwards-compat][upgrade-version][write-read-new-version]") {
```

Comment:
So... Backwards compatibility... Since those tests require access to old arrays that are available locally but not on s3, I decided not to port them in rest-CI at this moment at least. We could consider doing that implementing some kind of copy to s3 but it didn't seem like a priority to me given the effort needed..